apps/native-component-list/src/screens/Video/VideoScreen.tsx

```diff
@@ -49,6 +49,7 @@ const eventsToListen: (keyof VideoPlayerEvents)[] = [
   'volumeChange',
   'playToEnd',
   'sourceChange',
+  'progressUpdate',
```

Comment:
What do you all think about renaming it to just `progress`? This is how it's called on Web.

Reply:
```suggestion
  'timeUpdate',
```